powershell-yaml.psm1

```diff
@@ -235,14 +235,17 @@ function ConvertFrom-Yaml {
     }
 }
 
+
 function ConvertTo-Yaml {
     [CmdletBinding()]
     Param(
-        [Parameter(Mandatory=$false,ValueFromPipeline=$true)]
+        [Parameter(ValueFromPipeline=$true)]
         [System.Object]$Data,
-        [Parameter(Mandatory=$false)]
+        
         [string]$OutFile,
-        [switch]$JsonCompatible=$false,
+        
+        [YamlDotNet.Serialization.SerializationOptions]$Options = [YamlDotNet.Serialization.SerializationOptions]::Roundtrip,
```

Comment:
Current Default behavior is ```EmitDefaults```. I would like to preserve that. People updating from oldr releases, should not have to change their code.

Reply:
I have implemented parameter sets. The old behavior is still there and one can use the options parameter to have more control.